src/styles/calendar__day.scss

```diff
@@ -0,0 +1,25 @@
+.calendar__day {
```

Comment:
Add '.calendar' as a parent selector to '.calendar__day' to apply BEM methodology. So, no needs to create an own file for calendar__day